### PR TITLE
Update repository add documentation

### PR DESCRIPTION
**Pull request by t-mon**

### CODE REVIEW

File: src/pages/documentation/users/installation/core.md [hunk 1]
```diff
@@ -84,16 +86,16 @@ For each of the repositories four architectures are provided:
...
 ```bash
-deb http://repository.nymea.io <distro> main
+deb [arch=amd64,arm64,armhf signed-by=/etc/apt/trusted.gpg.d/nymea.gpg] http://repository.nymea.io <distro> main
 ```
 
 Add the repository signing key by running:
 
 ```bash
-sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-key A1A19ED6
+sudo gpg --homedir /tmp --no-default-keyring --keyring /etc/apt/trusted.gpg.d/nymea.gpg --keyserver keyserver.ubuntu.com --recv-keys A1A19ED6
```
Comment (mzanetti) [inline src/pages/documentation/users/installation/core.md:98]: I don't think this is the proper way...

Comment (t-mon) [reply src/pages/documentation/users/installation/core.md]: updated

File: src/pages/documentation/developers/build-env.md [hunk 2]
```diff
@@ -8,10 +8,29 @@ title: Build environment
 Like with a regular nymea installation, all the required packages can be obtained from the nymea repository. In addition to the
 regular repository.
 
+
+```bash
+echo "deb http://repository.nymea.io $(lsb_release -s -c) main" | sudo tee /etc/apt/sources.list.d/nymea.list
+```
+
+In order to make sure you get our packages, add the public repository singing key by running:
```
Comment (mzanetti) [inline src/pages/documentation/developers/build-env.md:16]: The packages in the nymea repository are signed with nymeas GPG key which can be imported by running:

File: src/pages/documentation/developers/build-env.md [hunk 3]
```diff
@@ -8,10 +8,29 @@ title: Build environment
...
+
+```bash
+sudo wget -O /etc/apt/trusted.gpg.d/nymea.gpg https://repository.nymea.io/nymea.gpg
+```
+
+You can verify the fingerprint of your downloaded key file with following output:
```
Comment (mzanetti) [inline src/pages/documentation/developers/build-env.md:22]: Optionally, the keys fingerpring can be verified with:

**Review by mzanetti — COMMENTED** [review]
Better, thanks!
I've added some suggestions for the wording.

File: src/pages/documentation/users/installation/core.md [hunk 4]
```diff
@@ -81,19 +95,31 @@ For each of the repositories four architectures are provided:
 * amd64
 * armhf
 * arm64
+* riscv64
```
Comment (mzanetti) [inline src/pages/documentation/users/installation/core.md:98]: Just as a side note: riscv64 is currently only supported for ubuntu jammy and kinetic.
There is no debian stable for riscv64 yet (likely bookworm will include that thought).

**Review by mzanetti — APPROVED** [review]

File: src/pages/documentation/developers/build-env.md [hunk 5]
```diff
@@ -8,10 +8,28 @@ title: Build environment
 Like with a regular nymea installation, all the required packages can be obtained from the nymea repository. In addition to the
 regular repository.
 
+
+```bash
+echo "deb http://repository.nymea.io $(lsb_release -s -c) main" | sudo tee /etc/apt/sources.list.d/nymea.list
```
Comment (mzanetti) [inline src/pages/documentation/developers/build-env.md:13]: Hmmm... as someone on discord just pointed out, this is missing the the `deb-src` line which previously was added using the `-s` switch in apt-add-repository.

**Review by mzanetti — CHANGES_REQUESTED** [review]

File: src/pages/documentation/developers/build-env.md [hunk 6]
```diff
@@ -8,10 +8,28 @@ title: Build environment
...
+
+Optionally, the keys fingerprint can be verified with:
+```bash
+gpg --show-keys --with-fingerprint /etc/apt/trusted.gpg.d/nymea.gpg
+```
+
 ```bash
-sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-key A1A19ED6
-sudo apt-add-repository -s "deb http://repository.nymea.io `lsb_release -cs` main"
-sudo apt-get update
```
Comment (mzanetti) [inline src/pages/documentation/developers/build-env.md:14]: Also the apt-get update step is missing now, which that user ran into and failed to installed build-deps...

**Review by mzanetti — CHANGES_REQUESTED** [review]

File: src/pages/documentation/developers/build-env.md (hunk 6 above)
Comment (t-mon) [reply src/pages/documentation/developers/build-env.md:14]: fixed, was a copy past error...

File: src/pages/documentation/users/installation/app.md [hunk 7]
```diff
@@ -63,19 +63,38 @@ For each of the repositories the following architectures are provided:
 * amd64
 * armhf
 * arm64
+* riscv64
 
-To enable the repository, create a file named `/etc/apt/sources.list.d/nymea.list` with the following content:
+To enable the repository, create a file named `/etc/apt/sources.list.d/nymea.list` using following command:
```
Comment (mzanetti) [inline src/pages/documentation/users/installation/app.md:68]: sorry, another small issue. there's a "the" missing now.

**Pull request merged**